ml_engine/online_prediction/README.md

```diff
@@ -0,0 +1,38 @@
+# Online Prediction with the Cloud Machine Learning Engine
```

Comment:
We don't have hand-written readmes in here any more. Please move all of this to the documentation and just link to the docs from here. I can add an auto-generated readme later.